src/lib/RequestTypes.ts

```diff
@@ -203,7 +203,7 @@ export class AccountsRequest {
                 return {
                     kind: RequestType.SIGNTRANSACTION,
```

Comment:
Should be `RequestType.SIGN_TRANSACTION` everywhere. (cf. "RequestType.EXPORT_WORDS")